compiler+runtime/test/jank/form/def/pass-meta.jank

```diff
@@ -8,4 +8,14 @@
 (assert (= one 'one))
 (assert (= (:doc (meta #'one)) nil))
 
+; Analysis with global variables
```

Comment:
This is not how new tests should be added. Each test should have a dedicated test file, describing what it's testing. Also, we now have possibilities for extra errors, which come from analyzing meta. That needs to be tested.